starknet_py/hash/sierra_class_hash.py

```diff
@@ -0,0 +1,55 @@
+from typing import List
+
+from starkware.cairo.common.poseidon_hash import poseidon_hash_many
+
+from starknet_py.common import int_from_bytes
+from starknet_py.hash.utils import _starknet_keccak
+from starknet_py.net.client_models import NewContractClass, NewEntryPoint
+
+
+def compute_sierra_class_hash(sierra_contract_class: NewContractClass) -> int:
+    """
+    Calculate class hash of a NewContractClass.
+    """
+    sierra_version = int_from_bytes(
+        ("CONTRACT_CLASS_V" + sierra_contract_class.contract_class_version).encode(
+            "ascii"
+        )
+    )
```

Comment:
How do we know it has to have `"CONTRACT_CLASS_V"` prefix?

Reply:
I found [this line in cairo-lang](https://github.com/starkware-libs/cairo-lang/blob/master/src/starkware/starknet/core/os/contract_class/contract_class.py#L75). The function prepares a ContractClass struct for input to the function computing the class_hash in cairo.